spec/controllers/responses_controller_spec.rb

```diff
@@ -45,5 +45,10 @@
     it 'finishes the response' do
       expect { subject }.to change { Events::ResponseFinished.count }.by 1
     end
+
+    it 'sends the email' do
+      expect(ResponseMailer).to receive(:response_email).and_call_original
```

Comment:
Ik heb er een delivery test aan toegevoegd. Maar nu mag de invitations_controller_spec ook worden aangepast (daar had ik deze code min of meer van overgenomen).